examples/complete/main.tf

```diff
@@ -0,0 +1,44 @@
+module "vars" {
+  source = "../../modules/vars"
+
+  stack_config_local_path = var.stack_config_local_path
+  stack                   = var.stack
+  component_type          = var.component_type
+  component               = var.component
+
+  context = module.this.context
+}
+
+module "folders" {
+  source = "../../modules/create_folder"
+}
+
+
+# module "settings" {
```

Comment:
if the code block is not necessary, please remove it or have comment why you need it